--- cli-clipboard-0.4.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ cli-clipboard-0.4.0/Cargo.toml	2024-05-30T09:52:26.413462+00:00
@@ -21,7 +21,7 @@
 repository = "https://github.com/actuallyallie/cli-clipboard"
 
 [target."cfg(all(unix, not(any(target_os=\"macos\", target_os=\"android\", target_os=\"emscripten\"))))".dependencies.wl-clipboard-rs]
-version = "0.7"
+version = "0.8"
 
 [target."cfg(all(unix, not(any(target_os=\"macos\", target_os=\"android\", target_os=\"emscripten\"))))".dependencies.x11-clipboard]
-version = "0.7"
+version = "0.9"
